modules/apps/josh-lapurd-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,228 @@
+				</div>
+
+				<div class="row row2">
+					<div class="col-md-6 grid-left">
+						<h3>Bottom left Title</h3>
+						<p>
+						Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ac sapien auctor mi imperdiet faucibus quis eu risus. Nulla cursus ligula varius euismod pharetra. Aenean a vulputate enim, quis hendrerit metus. Sed ut sem non eros tincidunt congue viverra id diam. Aenean sed nibh nisi.
+						</p>
+					</div>
+					<div class="col-md-6 grid-right">
+						<h3>Bottom right Title</h3>
+						<p>
+						Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ac sapien auctor mi imperdiet faucibus quis eu risus. Nulla cursus ligula varius euismod pharetra.
+						</p>
+					</div>
+				</div>
+			</div>
+		</div>
+	</section>
+
+	<section class="bg-dark" id="highlights">
+		<div class="row">
+			<div class="theme-container">
+				<div class="row">
+					<div class="content-desc">
+						<a class="link-fade text-caps">This is a Title, some awesome title.</a>
+						<p>This is some pretty cool sub text. should be in white.</p>
+						<a class="link-button" href="">This is a link</a>
+					</div>
+					<div class="row">
+						<div class="gallery">
+							<ul>
+								<li>
+									<a alt="Gallery picture" href="">Zion National Park</a>
+									<img alt="Zion National Park" class="gall-image" id="gall-img-1" src="../images/gallery1.jpg"/>
+								</li>
+								<li>
+									<a alt="Gallery picture" href="">Badminton</a>
+									<img alt="Badminton" class="gall-image" id="gall-img-2" src="../images/gallery2.jpg"/>
+								</li>
+								<li>
+									<a alt="Gallery picture" href="">Snowboarding</a>
+									<img alt="Snowboarding" class="gall-image" id="gall-img-3" src="../images/gallery3.jpg"/>
+								</li>
+								<li>
+									<a alt="Gallery picture" href="">Rock Climbing</a>
+									<img alt="Rock Climbing" class="gall-image" id="gall-img-4" src="../images/gallery4.jpg"/>
+								</li>
+							</ul>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</section>
+
+	<section id="portletSect">
+		<#if selectable>
+			<@liferay_util["include"] page=content_include />
+		<#else>
+			${portletDisplay.recycle()}
+
+			${portletDisplay.setTitle(the_title)}
+
+			<@liferay_theme["wrap-portlet"] page="portlet.ftl">
+				<@liferay_util["include"] page=content_include />
+			</@>
+		</#if>
+	</section>
+
+	<!--======================================================== footer ========================================================-->
+	<hr/>
+	<footer id="footer" role="contentinfo">
+		<p class="powered-by text-center">
+			<@liferay.language key="powered-by" /> <a href="http://www.liferay.com" rel="external">Liferay</a>
+		</p>
+	</footer>
+
+</div>
+
+<@liferay_util["include"] page=body_bottom_include />
+<@liferay_util["include"] page=bottom_include />
+
+<!-- inject:js -->
+<!-- endinject -->
+	<script src="http://code.jquery.com/jquery-1.7.1.min.js" type="text/javascript"></script>
```

Comment:
jQuery is already provided by Liferay.  You should not have to bring it in again?